examples/compliance.mql.yaml

```diff
@@ -29,6 +29,18 @@ frameworks:
       title: Limit access to system configuration
     - uid: compliance-x-03
       title: Harden systems to security recommendations
+    - uid: compliance-x-04
+      title: Only use ciphers with sufficient bits
+  - title: exception-1 # something auto-generated
+    type: 4 # IGNORED/SNOOZED
```

Comment:
is it both or just one? would be nice to have 2 groups in here with the 2 examples

Reply:
It's just one. The type is actually called `IGNORE`, but at some places we use the word `SNOOZE`. As I understood, going forward `SNOOZE` will win.